web/flux_to_redux/simple/examples/influx_implementation/components/redux_big_block.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_redux.dart';
+
+import '../store.dart';
+
+part 'redux_big_block.over_react.g.dart';
+
+UiFactory<ReduxBigBlockProps> ConnectedReduxBigBlock = connect<FluxStore, ReduxBigBlockProps>(
+  mapStateToProps: (state) => (ReduxBigBlock()..backgroundColor = state.backgroundColor),
+  mapDispatchToProps: (dispatch) =>
+      (ReduxBigBlock()..changeBackgroundColor = () => dispatch(UpdateBackgroundColorAction())),
+  pure: false,
```

Comment:
@greglittlefield-wf @joebingham-wk 
So after a little testing, if `pure: false` is used the redux component will always re-render on any state change (similar to flux) BUT if you remove this arg and instead use `areStatesEqual: (_, __) => false,` it will not re-render on state changes that do not affect its props:

![influx_redux_pure](https://user-images.githubusercontent.com/41018730/71695860-a3728580-2d70-11ea-98bc-561eba528892.gif)

Reply:
I must be missing some code somewhere - would you mind pushing that change up to a branch? When I make the parameter change and update the component code to look like yours, my profiler still shows the component rendering

Reply:
oh i also removed the dispatch to props... because similar to what i mentioned for the flux action to props stuff, same thing applied here... it seems problematic to me and im not sure how to go about even using dispatchToProps too use the same instance.

Reply:
I removed that as well and am still not seeing what you are :/

Reply:
I added an additional prop to the middle component that toggles a state bool field and did not include that in the mapStateToProps for the redux component.

Reply:
Ah yep, that would do it!

Reply:
@greglittlefield-wf